src/components/dialogs/resubmit.vue

```diff
@@ -204,11 +223,12 @@ export default class ResubmitDialog extends Mixins(
       }
     }
 
-    const success = await this.createPayment({
+    // if resubmit succeeded then create the payment
+    success = success && await this.createPayment({
       action: PaymentAction.RESUBMIT,
       nrId: this.getNrId,
       filingType: FilingTypes.NM620,
-      priorityRequest: this.getPriorityRequest // same value as originally
+      priorityRequest: this.getPriorityRequest
     } as CreatePaymentParams, onSuccess)
```

Comment:
Let's see with Linda if we can do a `PaymentAction.CREATE` here instead. I think it would the best approach.

Reply:
Can we stay with both CREATE and RESUBMIT actions? The API can always combine them when it calls the Pay API.

Reply:
ok. Does it create a "NR L..." too, right? Same process?

Reply:
Yes, it creates a NRL (which doesn't live for very long as payment is forthcoming). Same process.

Reply:
ok 👍